.github/pull_request_template.md

```diff
@@ -19,6 +19,9 @@ Currently available teams are:
 If your PR doesn't fall into those categories please contact
 the full review team `@conda-forge/staged-recipes`.
 
+Once the PR is ready for review, please contact
+`@conda-forge/staged-recipes` to have it merged.
+
```

Comment:
Can we put in the admin command here?

I think it is `conda-forge-admin please ping staged-recipes` or something like that?